lib/http-context.js

```diff
@@ -192,27 +191,45 @@ function coerce(str) {
 // coerce every string in the given object / array
 function coerceAll(obj) {
   var type = Array.isArray(obj) ? 'array' : typeof obj;
+  var i;
+  var n;
 
   switch (type) {
     case 'string':
       return coerce(obj);
     case 'object':
       if (obj) {
-        Object.keys(obj).forEach(function(key) {
+        var props = Object.keys(obj);
+        for (i = 0, n = props.length; i < n; i++) {
+          var key = props[i];
           obj[key] = coerceAll(obj[key]);
-        });
+        }
       }
       break;
     case 'array':
-      obj.map(function(o) {
-        return coerceAll(o);
-      });
+      for (i = 0, n = obj.length; i < n; i++) {
+        coerceAll(obj[i]);
+      }
       break;
   }
 
   return obj;
 }
 
+function buildArgs(ctx, method, fn) {
+  var args = ctx.args || {};
```

Comment:
This assignment looks dead to me.  It's reassigned in the try block and it's unused in the catch block.